pip_compile/Tiltfile

```diff
@@ -0,0 +1,39 @@
+load("ext://uibutton", "cmd_button")
+
+
+SCRIPT = os.path.join(os.getcwd(), "scripts", "controller.py")
+DEFAULT_COMPILE_ARGS = ["--allow-unsafe", "--generate-hashes"]
+
+
+def pip_compile_button(
```

Comment:
When I try to start the postajob project with 

```
pip_compile_button("postajob")
```

in the Tiltfile, the app resources never come up and the controller seems to hang:

![image](https://github.com/DirectEmployers/tilt-extensions/assets/5957867/0f0013c8-c4c9-4c74-9052-0695acf11141)